extensions/extensions-helm/extensions-helm-hive/pom.xml

```diff
@@ -15,7 +15,6 @@
 
 	<modules>
         <module>aissemble-hive-metastore-db-chart</module>
-        <module>aissemble-hive-metastore-service-chart</module>
```

Comment:
I: Refactored the metastore service to the `spark-infrastructure` aggregator module.  Next few files reflect this change.

Reply:
A: I'd rather keep hive separate, since it's a standalone data warehousing solution outside of it's use with Spark.

Reply:
I can go either way on it.  I think it's an interesting case because while it absolutely can be used outside of spark, we're centralizing its usage to the spark infrastructure.  I agree that it's not immediately an intuitive sibling to SHS and STS, but given that the method of consumption through aissemble is exclusively in the context of spark (and, pointedly, through the newly consolidated spark-infrastructure chart) I feel a bit weird about treating the hive ecosystem (service + db) as a wholly separate entity in the baseline.  

As an idea, how would you feel about a directory structure where `extensions-spark-infrastructure` is parent to `extensions-hive` and `extensions-spark-services`-- The former housing `aissemble-hive-metastore-chart` and `aissemble-hive-mysql-db-chart`, while the latter houses the `STS` and `SHS` charts?

Reply:
I guess I'm worried it will encourage unnecessary coupling to Spark.  At the end of the day, where it's housed is transparent to downstream projects, so it's not a huge deal either way.  It's just about what we think will create the lowest friction for baseline developers.  If we regret the decision, it's easy enough to move again.

Reply:
I can see it either way, but agree it's low lift to revert if we don't like it.  I'll keep it refactored for now, and will plan to refactor metastore-db in the next ticket.  Like you said, if we hate it, we can move it again.